modules/preview-react/text/stories/PropTables.splitprops.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {TextProps, TypeLabelProps, TypeLevelProps} from '@workday/canvas-kit-preview-react/text';
+
+// <ArgsTable of={Banner} /> generates a very large props table given that
+// BannerProps includes FlexProps. Use this dummy component instead to
+// limit the props shown.
+export const TextComponent = (_: TextProps) => <div />;
+
+// <ArgsTable of={Banner.Icon} /> generates a props table with
+// SystemIcon props. Use this dummy component instead to limit the props shown.
+export const LabelComponent = (_: TypeLabelProps) => <div />;
+
+// <ArgsTable of={Banner.Label} /> generates a props table with
```

Comment:
remove?

Reply:
No, it's mock component to generate prop table in doc. But I need to update comments there.

Reply:
without mock component `ArgsTable` generates really big table with ALL possible props, not text only.

Reply:
ah I was referencing how is says `Banner.Label` which confused me

Reply:
I see. Fixed it 👍